index.js

```diff
@@ -4,10 +4,12 @@ const stringWidth = require('string-width');
 
 module.exports = (input, columns, opts) => {
 	opts = Object.assign({
-		position: 'end'
+		position: 'end',
+		preferTruncationOnSpace: false
 	}, opts);
 
-	const position = opts.position;
+	const {position} = opts;
+	const {preferTruncationOnSpace} = opts;
```

Comment:
You can also destructure multiple values. Like this: `const {position, preferTruncationOnSpace} = opts;`